### DISCUSSION TITLE
Can't add or Import QSO's

### DISCUSSION THREAD

**Discussion by KarlFrederik** [magicbug/Cloudlog#2222] [General]
Hello,
I've installed Cloudlog yesterday, and I've noticed that I can't add any QSO's or import .adi files.
This is the Error I get:
<img width="943" alt="Bildschirmfoto 2023-06-22 um 21 18 15" src="https://github.com/magicbug/Cloudlog/assets/42519173/45726962-1937-414a-8e4b-c25cf79e371c">
What could cause this error?
My System:
Ubuntu Server, Not installed via Docker
I've installed it via the Wiki.

**Comment by phl0** [comment]
What exactly does not work? Importing obviously fails due to lacking write permissions for the user the web server runs under. Check permissions for `uploads` folder.

**Comment by KarlFrederik** [comment]
I've configured the permissions exactly like the Wiki told me.
I've checked under what user Apache is running (www-data).
I've couldn't add any QSO's under "Live QSO" or "Post QSO" and can't Import any Files.
But now I've done chmod 660 for all folders the wiki told me to and it works now.

**Comment by KarlFrederik** [comment]
Nevermind, 
i can add Live QSO's, but not Post QSO's or import any Files.
